README.md

```diff
@@ -1,70 +1,234 @@
+![Screenshot (26)](https://github.com/Ellyboi/react-capstone-project/assets/70116262/c99fbfff-b5ea-4c2c-b713-531d492f61e3)
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+## Desktop View
 
-### `npm run eject`
+![Screenshot (27)](https://github.com/Ellyboi/react-capstone-project/assets/70116262/d795a345-bef9-470e-a78a-8ef84b0f99e9)
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+## Built With <a name="built-with"></a>
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+### Tech Stack <a name="tech-stack"></a>
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+<details>
+    <summary>Client</summary>
+    <ul>
+        <li><a href="https://es.react.dev/">React</a></li>
+        <li><a href="https://redux.js.org/">Redux</a></li>
+        <li><a href="https://jestjs.io/">Jest (Testing)</a></li>
+    </ul>
+</details>
 
-## Learn More
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+<!-- Features -->
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+### Key Features <a name="key-features"></a>
 
-### Code Splitting
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+- **[Home Page]**
+- **[Details Page]**
+- **[Search functionality]**
 
-### Analyzing the Bundle Size
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-### Making a Progressive Web App
+<!-- LIVE DEMO -->
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+## Live Demo <a name="live-demo"></a>
 
-### Advanced Configuration
+- [Live Demo](https://www.loom.com/share/df65b6d1292846c88af2370c5ea6f109)
 
```

Comment:
- [ ] This link is supposed to link to your live project. Your deployed project. But it redirects to your presentation video. Kindly add your deployed link instead.

### Demo Link
![demo-link](https://github.com/Ellyboi/react-capstone-project/assets/26597432/dc1daae7-1ee9-4e56-a192-3d980f68c77b)